src/bot/listeners/test-tensor.listener.js

```diff
@@ -217,7 +272,8 @@ class TestTensorListener {
         return next();
       }
 
-      if (ctx.from.id !== creatorId) {
+      if (ctx.from.id !== 143875991) {
```

Comment:
Please return creatorId. If you need to mock your creator id, change it into the creator file and don't commit it.